setup/setup.wxs

```diff
@@ -1,204 +1,204 @@
-    <!--<UIRef Id="WixUI_Advanced" />-->
-    <UIRef Id="WixUI_InstallDir" />
-
-    <!--Customisation of Features and install location-->
-    <UIRef Id="WixUI_ErrorProgressText" />
-    <WixVariable Id="WixUILicenseRtf" Value="EULA.rtf" />
-    <WixVariable Id="WixUIBannerBmp" Value="Banner.bmp" />
-  </Product>
-</Wix>
+<?xml version="1.0" encoding="UTF-8"?>
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Product Id="$(var.ProductId)" Name="$(var.ProductFullName)" Language="1033" Version="$(var.ProductVersion)" Manufacturer="$(var.Manufacturer)" UpgradeCode="35BAE78E-2BCE-491C-A496-450A82E2BB5E">
+    <Package InstallerVersion="200" Compressed="yes" InstallPrivileges="elevated" Platform="x64" />
+
+    <Condition Message="For a correct installation of the software, administrator rights are necessary. You do not have these rights. Please contact your IT-administrator to carry out the installation">Privileged</Condition>
+
+    <Upgrade Id="35BAE78E-2BCE-491C-A496-450A82E2BB5E">
+      <UpgradeVersion OnlyDetect="no" Property="PREVIOUSFOUND" Minimum="5.0.0" IncludeMinimum="yes" Maximum="$(var.ProductVersion)" IncludeMaximum="no" />
+    </Upgrade>
+
+    <!--Setup Properties -->
+    <Property Id="ALLUSERS" Value="1" />
+    <Media Id="1" Cabinet="media1.cab" EmbedCab="yes" />
+
+    <!--Programm Files Set by bundle Setup-->
+    <Directory Id="TARGETDIR" Name="SourceDir">
+      <Directory Id="OSP_SUITE" Name="$(var.Manufacturer)">
+        <Directory Id="INSTALLDIR" Name="$(var.ProductFullName)">
+          <Component Id="PKSim.exe" Guid="7E30E944-CA31-4330-93E7-A6F4F4AE4663" Win64="yes">
+            <File Name="PKSim.exe" Id="PKSim.exe" KeyPath="yes" Source="$(var.DeployDir)/PKSim.exe">
+              <Shortcut Name="$(var.ProductName)" Id="PKSimShortcut" Icon="PKSimIcon.exe" Directory="OSPSuiteMenuDir" WorkingDirectory="INSTALLDIR" Advertise="yes" />
+            </File>
+
+            <File Name="PKSim5FileIcon.ico" Id="PKSim5FileIcon.ico" Source="$(var.DeployDir)/PKSim5FileIcon.ico" />
+            <ProgId Id="PKSim.pksim5File" Description="PK-Sim Project File" Icon="PKSim5FileIcon.ico" IconIndex="0">
+              <Extension Id="pksim5" ContentType="application/pksim5">
+                <Verb Id="open" Command="Open" TargetFile="PKSim.exe" Argument='"%1"' />
+              </Extension>
+            </ProgId>
+
+            <File Name="JournalFile.ico" Id="JournalFile.ico" Source="$(var.DeployDir)/JournalFile.ico" />
+            <ProgId Id="OSPSuite.JournalFile" Description="OSPSuite Journal File" Icon="JournalFile.ico" IconIndex="0">
+              <Extension Id="sbj" ContentType="application/sbj">
+                <Verb Id="open" Command="Open" TargetFile="PKSim.exe" Argument='/j "%1"' />
+              </Extension>
+            </ProgId>
+
+            <RegistryValue Root="HKLM" Key="Software\$(var.Manufacturer)\PK-Sim\$(var.SuiteVersion)" Name="InstallPath" Type="string" KeyPath="no" Value="[INSTALLDIR]PKSim.exe" />

```

Comment:
@Yuri05 Installing under SuiteVersion (that comes from the script) e.g. 9 or 10 etc..